src/SFA.DAS.AssessorService.Data/StandardRepository.cs

```diff
@@ -248,15 +248,16 @@ public async Task<int> GetEpaoStandardsCount(string endPointAssessorOrganisation
             return @params.Get<int>("Count");
         }
 
-        public async Task<EpoRegisteredStandardsResult> GetEpaoRegisteredStandards(string endPointAssessorOrganisationId, int pageSize, int? pageIndex)
+        public async Task<EpoRegisteredStandardsResult> GetEpaoRegisteredStandards(
+            string endPointAssessorOrganisationId, int? pageSize, int? pageIndex)
         {
             var epoRegisteredStandardsResult = new EpoRegisteredStandardsResult
             {
                 PageOfResults = new List<EPORegisteredStandards>(),
                 TotalCount = 0
             };
             var total = await GetEpaoStandardsCount(endPointAssessorOrganisationId);
-            var skip = ((pageIndex ?? 1) - 1) * pageSize;
+            var skip = ((pageIndex ?? 1) - 1) * (pageSize ?? 10);
             var result = await _unitOfWork.Connection.QueryAsync<EPORegisteredStandards>(
                 "EPAO_Registered_Standards", 
                 param: new
```

Comment:
Need to apply the same pagesize logic to the `Take` param. Also, can we just apply the "Default to 10" here rather than checking for null throughout the code?